android/app/src/main/java/app/organicmaps/downloader/BottomPanel.java

```diff
@@ -103,29 +96,18 @@ public void update()
       {
         switch (status)
         {
-        case CountryItem.STATUS_UPDATABLE:
-          UpdateInfo info = MapManager.nativeGetUpdateInfo(root);
-          setUpdateAllState(info);
-          break;
-
-        case CountryItem.STATUS_DOWNLOADABLE:  // Special case for "Countries" node when no maps currently downloaded.
-        case CountryItem.STATUS_DONE:
-        case CountryItem.STATUS_PARTLY:
-          show = false;
-          break;
-
-        case CountryItem.STATUS_PROGRESS:
-        case CountryItem.STATUS_APPLYING:
-        case CountryItem.STATUS_ENQUEUED:
-          setCancelState();
-          break;
-
-        case CountryItem.STATUS_FAILED:
-          setDownloadAllState();
-          break;
-
-        default:
-          throw new IllegalArgumentException("Inappropriate status for \"" + root + "\": " + status);
+          case CountryItem.STATUS_UPDATABLE ->
+          {
+            UpdateInfo info = MapManager.nativeGetUpdateInfo(root);
+            setUpdateAllState(info);
+          }  // Special case for "Countries" node when no maps currently downloaded.
+          case CountryItem.STATUS_DOWNLOADABLE, CountryItem.STATUS_DONE, CountryItem.STATUS_PARTLY ->
+              show = false;
+          case CountryItem.STATUS_PROGRESS, CountryItem.STATUS_APPLYING, CountryItem.STATUS_ENQUEUED ->
+              setCancelState();
+          case CountryItem.STATUS_FAILED -> setDownloadAllState();
+          default ->
+              throw new IllegalArgumentException("Inappropriate status for \"" + root + "\": " + status);
```

Comment:
Do you have any limit set in your IDE? 120 chars in a line is ok, and it is not a hard limit. E.g. if it's 123 chars, then it's also ok if the code is readable.

```suggestion
          case CountryItem.STATUS_DOWNLOADABLE, CountryItem.STATUS_DONE, CountryItem.STATUS_PARTLY -> show = false;
          case CountryItem.STATUS_PROGRESS, CountryItem.STATUS_APPLYING, CountryItem.STATUS_ENQUEUED -> setCancelState();
          case CountryItem.STATUS_FAILED -> setDownloadAllState();
          default -> throw new IllegalArgumentException("Inappropriate status for \"" + root + "\": " + status);
```